plug-ins/sso/openid-connect/api/src/test/java/org/eclipse/kapua/plugin/sso/openid/exception/OpenIDErrorCodesTest.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.plugin.sso.openid.exception;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(JUnitTests.class)
+public class OpenIDErrorCodesTest extends Assert {
+
+    @Test
+    public void openIDErrorCodesEnumTest() {
```

Comment:
This is pretty much not useful, since.

Enum.name() will be always equals to the `enum` name

Reply:
The only reason for this test was to achieve better code coverage, which was our goal here. I have now deleted this.